wagtail_streamfield_migration_toolkit/test/tests/test_rename.py

```diff
@@ -0,0 +1,153 @@
+from django.test import TestCase, SimpleTestCase
+
+from wagtail_streamfield_migration_toolkit.utils import get_altered_raw_data
+from ..models import SampleModel
+
+
+class RenameUtilsTestCase(SimpleTestCase):
+    @staticmethod
+    def make_json_data():
+        stream_data = [
+            {
+                "type": "field1",
+                "value": "Simple Rename Value",
+            },
+            {
+                "type": "field2",
+                "value": "This shouldn't change",
+            },
+            {
+                "type": "struct1",
+                "value": {
+                    "struct1field1": "Struct field rename",
+                    "struct1field2": "This shouldn't change",
+                },
+            },
+            {
+                "type": "stream1",
+                "value": [
+                    {"type": "stream1field1", "value": "Nested Stream field rename"},
+                    {"type": "stream1field2", "value": "This shouldn't change"},
+                ],
+            },
+        ]
+        return stream_data
+
+
+class RenameRawDataTestCase(TestCase):
```

Comment:
While these tests are good, they're a bit like integration tests, as we're creating a model instance and interacting with its API to retrieve the stream data before operating on it. I think it would be good to add some unit tests in which we create (in the local scope of the test methods) just the data structure for the block we want to test, and assert that the rename operation produces the correct output. This way there's no dependency on the way that we get the data from the model instance.

By only creating the blocks that we need in the context of a test it will also be a bit easier to keep track of things once we start dealing with nested blocks.